matchbox/models/models.py

```diff
@@ -95,7 +95,7 @@ def set_from_model_meta(self, model_meta):
         pk = fields.IDField()
         pk.contribute_to_class(cls, 'id')
 
-        setattr(cls, 'path', (cls._meta.collection_name, ))
+        cls.path = (cls._meta.collection_name, )
```

Comment:
Is there any reason to change this ? I think this work in the same way

Reply:
I prefer this way because it makes it easier for an IDE to recognize this as a variable. Aside from that, I prefer this style. If you'd like me to revert this, I will.

Reply:
ok this is good reason for change it in this way